test/dry_container_test.rb

```diff
@@ -21,5 +21,5 @@ class Create < Trailblazer::Operation
   # also allows our own options PLUS containers.
   it { Create.({}, {"model" => String}, my_container)["model"].must_equal String }
   it { Create.({}, {"model" => String}, my_container)["user_repository"].must_equal Object }
-  it { Create.({}, {"user_repository" => Integer}, my_container)["user_repository"].must_equal Integer }
+  it { Create.({}, my_container, {"user_repository" => Integer})["user_repository"].must_equal Integer }
```

Comment:
I am still unsure if we should keep supporting that API - I'd rather require users to build a contain "chain" themselves and allow only one argument.

Reply:
Hmm right.

Reply:
_().must_equal

Reply:
This has been removed @seuros ⚔️